src/schema/v2/me/conversation/index.ts

```diff
@@ -134,6 +134,18 @@ const ConversationItem = new GraphQLObjectType<any, ResolverContext>({
     permalink: {
       type: GraphQLString,
     },
+    liveItem: {
+      type: ConversationItemType,
+      resolve: (parent, _args, { artworkLoader }) => {
+        if (parent.item_type === "Artwork") {
```

Comment:
I think `parent` here is a conversation, so maybe what we are looking for is the individual item from `parent.items`?

Reply:
@starsirius I thought the naming convention here was to call this parameter `parent`. Perhaps I should change it to `item` because it is of type `ConversationItemType`. Additionally, I `console.log` out the object to double check. Also the conversation object doesn't have the property `item_type`.

Reply:
Ohh I think you're right. It is a conversation item (now I expand the diff 😅 ), instead of a conversation.